cluster/src/main/java/com/hartwig/pipeline/metadata/ApiRunStatus.java

```diff
@@ -0,0 +1,54 @@
+package com.hartwig.pipeline.metadata;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.hartwig.api.RunApi;
+import com.hartwig.api.model.Run;
+import com.hartwig.api.model.RunFailure;
+import com.hartwig.api.model.Status;
+import com.hartwig.api.model.UpdateRun;
+import com.hartwig.pipeline.execution.PipelineStatus;
+import com.hartwig.pipeline.jackson.ObjectMappers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ApiRunStatus {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ApiRunStatus.class);
+    private static final String PIPELINE_SOURCE = "Pipeline";
+    private static final String HEALTH_CHECK = "HealthCheck";
+
+    static void start(final RunApi runApi, final Run run) {
+        UpdateRun update = new UpdateRun().failure(null).status(Status.PROCESSING).startTime(timestamp());
+        try {
+            LOGGER.info("Updating API with run status and result [{}]", ObjectMappers.get().writeValueAsString(update));
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
```

Comment:
This is a bit of a funny one, maybe we should write a log message explaining the stacktrace too?